### PR TITLE
Add active_validation parameter to CommandTable

### PR DESCRIPTION
**Pull request by markheik**
Description:

Add active_validation parameter to CommandTable

By disabling the active validation in large command tables (e.g 1024 table entries), the speed improvements are up to 8x.

Fixes issue: #

Checklist:

- [x] Add tests for the change to show correct behavior.
- [x] Add or update relevant docs, code and examples.
- [x] Update CHANGELOG.rst with relevant information and add the issue number.
- [x] Add .. versionchanged:: where necessary.

### CODE REVIEW

File: src/zhinst/toolkit/command_table.py [hunk 1]
```diff
@@ -40,14 +40,36 @@ class ParentNode:
...
+    ):
         self._schema = schema
         self._path = path
         self._childs: t.Dict[t.Union[str, int], t.Any] = {}
+        self._active_validation = active_validation
 
     def __repr__(self) -> str:
         return "/" + "/".join(self._path)
 
+    def _validate_instance(self, instance, schema):
```
Comment (tobiasah) [inline src/zhinst/toolkit/command_table.py:54]: Missing docstring, typehints

File: src/zhinst/toolkit/command_table.py [hunk 2]
```diff
@@ -40,14 +40,36 @@ class ParentNode:
...
+    @property
+    def active_validation(self) -> bool:
+        """State of active validation.
+
+        Returns:
+            True if active validation is enabled.
+
+        .. versionadded:: 0.4.4
+        """
+        return self._active_validation
```
Comment (tobiasah) [inline src/zhinst/toolkit/command_table.py:67]: Missing test for `active_validation` property

File: src/zhinst/toolkit/command_table.py [hunk 3]
```diff
@@ -67,8 +89,10 @@ class ParentEntry(ParentNode):
         path: Path representation of the node.
```
Comment (tobiasah) [inline src/zhinst/toolkit/command_table.py:89]: Missing docstring for new argument `active_validation`

File: src/zhinst/toolkit/command_table.py [hunk 4]
```diff
@@ -40,14 +40,36 @@ class ParentNode:
...
```
Comment (tobiasah) [inline src/zhinst/toolkit/command_table.py:40]: Missing docstring for new argument `active_validation`

File: src/zhinst/toolkit/command_table.py [hunk 5]
```diff
@@ -223,8 +251,14 @@ class HeaderEntry(ParentEntry):
         version: JSON schema version
```
Comment (tobiasah) [inline src/zhinst/toolkit/command_table.py:251]: Missing docstring for new argument `active_validation`

File: src/zhinst/toolkit/command_table.py [hunk 6]
```diff
@@ -257,43 +291,111 @@ def _derefence_json(schema: t.Union[str, dict]) -> t.Any:
...
+    @property
+    def active_validation(self) -> bool:
+        """State of active validation.
+
+        Returns:
+            True if active validation is enabled.
+
+        .. versionadded:: 0.4.4
+        """
+        return self._active_validation
```
Comment (tobiasah) [inline src/zhinst/toolkit/command_table.py:384]: Missing Unittest for `active_validation` property

**Review by ghost — CHANGES_REQUESTED** [review]
Could you also update the Command Table section in the AWG example? (https://github.com/zhinst/zhinst-toolkit/blob/main/examples/awg.md#command-table) and explain in detail when the active validation should be disabled?

**Review by ghost — APPROVED** [review]
Pls don't forget to squash 😇

**Pull request merged**